Modelica/Magnetic/FluxTubes/Material/SoftMagnetic/PureIron/VacoferS2.mo

```diff
@@ -1,12 +1,14 @@
 within Modelica.Magnetic.FluxTubes.Material.SoftMagnetic.PureIron;
 record VacoferS2 "VACOFER S2 (99.95% Fe)"
   extends FluxTubes.Material.SoftMagnetic.BaseData(
+    label="Vacofer S2",
```

Comment:
```suggestion
    label="VACOFER S2",
```
See https://vacuumschmelze.com/Company/Quality-and-Safety/Information-for-Manufactured-Articles/VACOFER